nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-resources/pom.xml

```diff
@@ -252,6 +252,8 @@
         <nifi.analytics.connection.model.implementation>org.apache.nifi.controller.status.analytics.models.OrdinaryLeastSquares</nifi.analytics.connection.model.implementation>
         <nifi.analytics.connection.model.score.name>rSquared</nifi.analytics.connection.model.score.name>
         <nifi.analytics.connection.model.score.threshold>.90</nifi.analytics.connection.model.score.threshold>
+        <nifi.analytics.prediction.parallel.processing.timeout>60 secs</nifi.analytics.prediction.parallel.processing.timeout>
```

Comment:
I would avoid adding new properties for this. Instead, recommend just hard-coding the number of threads to some value like 4. And wait up to `nifi.web.request.timeout` amount of time.
I can't see anyone really having a need to configure these properties, and the more that we add, the more overwhelming it is to configure.